src/jira/client/index.ts

```diff
@@ -254,13 +254,13 @@ const batchedBulkUpdate = async (
 	data,
 	instance: AxiosInstance,
 	installationId: number,
-	logger:Logger,
+	logger?: Logger,
```

Comment:
Why should the logger be optional here when it wasn't optional before?

Reply:
Because the logger being passed into it from `JiraClient` is optional as well.  there were some situations where it would be undefined.